arionBackend/transformation/transformer.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABCMeta, abstractmethod
+
+
+class Transformer(metaclass=ABCMeta):
+    """
+    This class is an abstract base class for transformers of type "event language" -> "hierarchy".
+    """
+
+    @abstractmethod
+    def transform(self, queries):
+        """
+        This abstract method transforms the queries into the hierarchy.
+        :param queries: The queries to put into a hierarchy. 

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS1131)